src/qt/forms/sendcoinsdialog.ui

```diff
@@ -1108,10 +1108,10 @@
            <item>
             <widget class="QCheckBox" name="optInRBF">
              <property name="text">
-              <string>Request Replace-By-Fee</string>
+              <string>Allow Fee Increase</string>
```

Comment:
not exactly sure how it works in English, but shouldn't "fee increase" be lowercased? Capitalizing made sense for "Replace-By-Fee" since it is more of a name, but now we're using generic nouns

Reply:
The UI uses a mix of "Upper lower case" and "All Upper Case". For example "Custom change address", "Subtract fee from amount" vs. "Pay To", "Coin Control Features". In addition some sentences end with a period, others don't.

I'm happy to change it either way, but it might be better to have a single pull request to implement (and document) a consistent style across the wallet.

Reply:
:+1: lower case.